tools/pipeline_perf_test/load_generator/loadgen.py

```
@@ -355,15 +358,17 @@ def syslog_tcp_worker_thread(self, thread_id: int, args: dict) -> None:
 
+        print(f"Thread {thread_id}: Using TCP transport to syslog server {syslog_server}:{syslog_port}")
+
         # Create TCP socket for syslog
         sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
         sock.settimeout(5)
 
         try:
             sock.connect((syslog_server, syslog_port))
-            print(f"Thread {thread_id}: Successfully connected to syslog server {syslog_server}:{syslog_port}")
+            print(f"Thread {thread_id}: TCP connection established to {syslog_server}:{syslog_port}")
         except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)